validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -26,14 +26,14 @@ type grpcValidatorClient struct {
 }
 
 func (c *grpcValidatorClient) Duties(ctx context.Context, in *ethpb.DutiesRequest) (*ethpb.ValidatorDutiesContainer, error) {
-	dutiesResponse, err := c.beaconNodeValidatorClient.GetDuties(ctx, in)
+	dutiesResponse, err := c.beaconNodeValidatorClient.GetDutiesV2(ctx, in)
```

Comment:
Can you send the client and server updates in different PRs?

We will need the validator client to fallback to the old GetDuties endpoint, if V2 is not available in the beacon node yet.